wagtail/wagtailcore/query.py

```diff
@@ -197,12 +197,12 @@ def not_public(self):
         """
         return self.exclude(self.public_q())
 
-    def search(self, query_string, fields=None, backend='default'):
+    def search(self, query_string, fields=None, operator='or', backend='default'):
```

Comment:
Presumably this should be `operator=None` now, so that it defaults to the backend's own default operator?